libqtile/manager.py

```diff
@@ -177,6 +179,14 @@ def __init__(self, config, displayName=None, fname=None, no_spawn=False, state=N
         for i in self.groups:
             self.groupMap[i.name] = i
 
+        for grp in self.config.groups:
+            if isinstance(grp, ScratchPadConfig):
```

Comment:
You will just have more than one ScratchPad groups.
Since each ScratchPad and its DropDown objects are identified by name there is no problem with this. However, in current stage of expansion there is no benefit of having more than one.